i18n/en-US.yml

```diff
@@ -384,7 +384,7 @@ components:
       body: >-
         No transit connection was found between your origin and destination on
         the selected day of service, using the vehicle types you selected.
-      header: No transit connections
+      header: No itineraries found
```

Comment:
Do the other languages need to be updated too?

Reply:
I think so!